dialogflow/stopwatch/Stopwatch/AudioController.swift

```diff
@@ -37,7 +37,11 @@ class AudioController {
 
     let session = AVAudioSession.sharedInstance()
     do {
-      try session.setCategory(AVAudioSessionCategoryRecord)
+      if #available(iOS 10.0, *) {
+        try session.setCategory(AVAudioSession.Category.record, mode: AVAudioSession.Mode.spokenAudio)
+      } else {
+        // Fallback on earlier versions
```

Comment:
Should we do more to handle this or throw an error or cancel here?

Reply:
We are not handling the catch block, We can ignore this.